README.md

```diff
@@ -75,11 +75,12 @@ If we would like to:
 without touching the testartifact then we could set up the `actions.txt` (the config of the trace agent) like this:
 
 ```
-elapsed_time_in_nano net.test.TestClass test
-elapsed_time_in_ms net.test.TestClass2nd anotherMethod
-stack_trace net.test.TestClass2nd anotherMethod
-trace_args net.test.TestClass2nd methodWithArgs
-trace_retval net.test.TestClass2nd methodWithArgs
+elapsed_time_in_nano net.test.TestClass test params
+elapsed_time_in_ms net.test.TestClass2nd anotherMethod params
+stack_trace net.test.TestClass2nd anotherMethod params
+trace_args net.test.TestClass2nd methodWithArgs params
+trace_retval net.test.TestClass2nd methodWithArgs params
+counter net.test.TestClass2nd methodToCounter params
```

Comment:
In this introduction part this would be a working example to satisfy the requirements specified a few lines before.
So please revert this lines to make the example correct.

Reply:
Sure, I will revert back the changes.